Update Translation to include "Tip" flarum-lang (#348)
flarum/docs: docs/extend/language-packs.md

::: tip

> The Flarum Foundation have created the "flarum-lang" organization to support translators and ensure continuous availability of language packs. You can learn all about this by visiting the [GitHub repository](https://github.com/flarum-lang/about). If you want to support an existing language pack or start a new translation, it's best to [get in touch with the flarum-lang team](https://discuss.flarum.org/d/27519-the-flarum-language-project).

Reviewer: ```suggestion
The Flarum Foundation has created the "flarum-lang" organization to support translators and ensure continuous availability of language packs. You can learn more about this by [visiting the GitHub repository](https://github.com/flarum-lang/about). If you want to support an existing language pack or start a new translation, it's best to [get in touch with the `flarum-lang` team](https://discuss.flarum.org/d/27519-the-flarum-language-project).
```

> The Flarum Foundation is supporting translators while ensuring continous availibilty of language packs. Therefor "flarum-lang" was initiated. You can learn all about this by visiting the [GIT repository](https://github.com/flarum-lang/about). Either if you want to support an existing language pack - or if you want to start a new translation it's best to [get in touch with the flarum-lang team](https://discuss.flarum.org/d/27519-the-flarum-language-project). 

Reviewer: ```suggestion
The Flarum Foundation have created the "flarum-lang" organization to support translators and ensure continuous availability of language packs. You can learn all about this by visiting the [GitHub repository](https://github.com/flarum-lang/about). If you want to support an existing language pack or start a new translation, it's best to [get in touch with the flarum-lang team](https://discuss.flarum.org/d/27519-the-flarum-language-project).
```
